Sources/MongoSwift/ChangeStream.swift

```diff
@@ -3,22 +3,29 @@ import mongoc
 /// `ChangeStreamOptions` to resume or start a change stream where a previous one left off.
 /// - SeeAlso: https://docs.mongodb.com/manual/changeStreams/#resume-a-change-stream
 public struct ResumeToken: Codable, Equatable {
+    /// A `Document` associated with the most recent event seen by this change stream.
```

Comment:
This docstring doesn't actually need to be added ("most recent event seen by this change stream" doesn't apply outside the context of a change stream, and this docstring is on the resume token struct which isn't tied to a stream).